content/gui/xml/ingame/tabwidget/overview/overview_firestation.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
```

Comment:
Can the firestation tab and the doctor tab be abstracted into a single tab definition?